### PR TITLE
feat: add auth args when helm install/upgrade

### LINKED ISSUE #114: helm install/upgrade should add auth args

### PR DESCRIPTION
**Pull request by zqq454224016**
#### What this PR does / why we need it
feat: add auth args when helm install/upgrade

#### Which issue(s) this PR fixes

Fixes #114

### CODE REVIEW

File: pkg/helm/helm.go [hunk 1]
```diff
@@ -39,6 +39,8 @@ import (
 	"helm.sh/helm/v3/pkg/release"
 	"helm.sh/helm/v3/pkg/repo"
 	"helm.sh/helm/v3/pkg/storage/driver"
+	v1 "k8s.io/api/core/v1"
```
Comment (bjwswang) [inline pkg/helm/helm.go:42]: use `corev1` here

File: pkg/helm/helm.go [hunk 2]
```diff
@@ -94,8 +96,15 @@ func (h *HelmWrapper) install(ctx context.Context, logger logr.Logger, cli clien
 	i.Verify = false // TODO enable these args after we can config keyring
 	i.RepoURL = repo.Spec.URL
 	i.Keyring = "" // TODO enable these args after we can config keyring
-	// i.Username // FIXME
-	// i.Password // FIXME
+	if repo.Spec.AuthSecret != "" {
+		S := v1.Secret{}
```
Comment (bjwswang) [inline pkg/helm/helm.go:100]: avoid using a single uppercase character  `S` as the variable name.

Comment (bjwswang) [inline pkg/helm/helm.go:100]: avoid using a single uppercase character  `S` as the variable name.

**Review by bjwswang — APPROVED** [review]

**Comment by bjwswang** [comment]
@zqq454224016 after merged this pr,pls update the kubebb official document

**Comment by Abirdcfly** [comment]
website update is track by kubebb/website#41, example-test is track by #228

**Review by Abirdcfly — APPROVED** [review]

**Comment by bjwswang** [comment]
@zqq454224016  Thanks! Seems basic auth fully enabled in kubebb.

**Pull request merged**